juriscraper/pacer/email.py

```diff
@@ -0,0 +1,50 @@
+import datetime
+import re
+
+from .reports import BaseReport
+
+
+class NefEmail(BaseReport):
+    def __init__(self, court_id):
+        self.court_id = court_id
+
+    @property
+    def data(self):
+        base = {
+            "court_id": self.court_id,
+        }
+        parsed = {}
+        if self.tree is not None:
+            parsed = {
+                "case_name": self._get_case_name(),
+                "docket_number": self._get_docket_number(),
+                "date_filed": self._get_date_filed(),
+            }
+
+        return {**base, **parsed}
```

Comment:
Started to make the change, but I am not sure if that's the case. Here is the change I could do:

```Python
return {**(base or {}), **(parsed or {})}
````

But since both base and parsed are defined above the conditional, I think it should be good as is.

Reply:
Yeah, sorry, I misread. Looks like you made this change though. Mind removing it? I spent 30s trying to figure out what the `or`'s were doing only to realize the answer was nothing (at the moment).

Reply:
No problem. Will go back. Sorry about that.